packages/trader/src/App/Components/Elements/market-countdown-timer.jsx

```diff
@@ -109,14 +109,19 @@ const MarketCountdownTimer = ({
     Object.keys(time_left).forEach(interval => {
         if (interval === 'days') {
             if (time_left.days) {
-                timer_components += `${time_left.days} ${localize(time_left.days > 1 ? 'days' : 'day')}`;
+                const days_text = time_left.days > 1 ? 'days' : 'day';
```

Comment:
Improvement, we use this state, 3 times so probably better to create a func for it. like `const Pluralize = (name, num) => '${name}${count > 1 ? : 's' : ''}`